From 28c5458f4be35b5c7747c575595baa62df64759d Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:07:54 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit 6cc8bab6eaa59c423dfa09a95f07f8f0bd95469d
Merge: fb6b30887 9b9d485f3
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jul 8 03:07:00 2018 +0000

    Snap for 4880674 from 9b9d485f36af50430bc013884c19e79f4b19c7a1 to pi-dr1-release

    Change-Id: I964e17d0977d2435a83a7da96dc1fddaff472bf7

commit 9b9d485f36af50430bc013884c19e79f4b19c7a1
Author: Bill Yi <byi@google.com>
Date:   Wed Jul 4 07:57:23 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I1069f843cfa8d4d34feae318e8e70472a3539a96

commit fb6b30887edbb8ecf76242c5200782561d45b32b
Merge: e4a478d98 355856995
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:00:58 2018 +0000

    Snap for 4859140 from 355856995dc76463e4f4f61b8800503eda7aba09 to pi-dr1-release

    Change-Id: I4cbd13cd0f8d0bb94b4fb948a9758c77a570e15d

commit 355856995dc76463e4f4f61b8800503eda7aba09
Merge: 889e4712d 6bd0b845e
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Thu Jun 21 18:14:23 2018 +0000

    Merge "Import translations. DO NOT MERGE" into pi-dev

commit 6bd0b845ec5324dd9f769ab40bf4a009ffb3fba7
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 09:03:17 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I1ae1a7a31116d0c95bd27df4070fd72401f74032

commit e4a478d98bd8b302d644f7cc7c524d9f52256748
Merge: 38da7d418 889e4712d
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Thu Jun 21 03:06:14 2018 +0000

    Snap for 4853718 from 889e4712ddc12aebd56ee4f357a7b1c7324c8717 to pi-dr1-release

    Change-Id: I97988c2a1383db4d1178a9ea8ab038890d21d4d5

commit 889e4712ddc12aebd56ee4f357a7b1c7324c8717
Author: Bill Yi <byi@google.com>
Date:   Wed Jun 20 14:35:07 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I17f6549c3bbe95de5df1e88a5a3a5cdf3904c5ee

commit 38da7d4185842b75c745b63a7ecaf4814a07d678
Merge: 0e894a09a 4a15b3999
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jun 5 03:02:44 2018 +0000

    Snap for 4820872 from 4a15b3999291e108a43137c500b286f4c1c1fe87 to pi-dr1-release

    Change-Id: I10d4f4087c40f547b436dc048fd5c3a4bd6651f3

Change-Id: I91c0bf5208902bf1d305d68a5a0e5da9e968dccd
---
 res/values-am/strings.xml |  2 +-
 res/values-da/strings.xml | 76 +++++++++++++++++++--------------------
 res/values-iw/strings.xml |  4 +--
 res/values-ne/strings.xml |  4 +--
 res/values-sw/strings.xml |  6 ++--
 5 files changed, 46 insertions(+), 46 deletions(-)

diff --git a/res/values-am/strings.xml b/res/values-am/strings.xml
index 2e9a0608e..89ed1cd4f 100644
--- a/res/values-am/strings.xml
+++ b/res/values-am/strings.xml
@@ -253,7 +253,7 @@
     <string name="permission_explanation_header" msgid="5739405825039695327">"ታሪክዎን አብረው ይመልከቱ"</string>
     <string name="permission_explanation_subheader_calendar_and_SMS" msgid="5411144298889835768">"ኩነቶች እና መልዕክቶች"</string>
     <string name="permission_explanation_subheader_calendar" msgid="8785323496211704613">"ኩነቶች"</string>
-    <string name="permission_explanation_subheader_SMS" msgid="1904552086449525567">"መልእክቶች"</string>
+    <string name="permission_explanation_subheader_SMS" msgid="1904552086449525567">"መልዕክቶች"</string>
     <string name="hamburger_feature_highlight_header" msgid="5762679188565585072">"ጥቆማዎች"</string>
     <string name="hamburger_feature_highlight_body" msgid="4403398269286898733">"እውቂያዎችዎን የተደራጁና ጠቃሚ እንደሆኑ ያቆዩ"</string>
     <string name="undo" msgid="1425165101664071422">"ቀልብስ"</string>
diff --git a/res/values-da/strings.xml b/res/values-da/strings.xml
index d5e040339..0d8b2ecef 100644
--- a/res/values-da/strings.xml
+++ b/res/values-da/strings.xml
@@ -18,15 +18,15 @@
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="applicationLabel" msgid="3906689777043645443">"Kontaktpersoner"</string>
     <string name="contactsList" msgid="8661624236494819731">"Kontaktpersoner"</string>
-    <string name="shortcut_add_contact" msgid="7476283631356909204">"Tilføj kontaktperson"</string>
+    <string name="shortcut_add_contact" msgid="7476283631356909204">"Tilføj kontakt"</string>
     <string name="shortcutContact" msgid="749243779392912958">"Kontakt"</string>
     <string name="shortcutDialContact" msgid="746622101599186779">"Direkte opkald"</string>
     <string name="shortcutMessageContact" msgid="2460337253595976198">"Direkte besked"</string>
-    <string name="shortcutActivityTitle" msgid="1504383567778497348">"Vælg kontaktperson"</string>
-    <string name="contactInsertOrEditActivityTitle" msgid="6527505405325862674">"Føj til kontaktperson"</string>
-    <string name="contactPickerActivityTitle" msgid="4301062192337417640">"Vælg en kontaktperson"</string>
+    <string name="shortcutActivityTitle" msgid="1504383567778497348">"Vælg kontakt"</string>
+    <string name="contactInsertOrEditActivityTitle" msgid="6527505405325862674">"Føj til kontakt"</string>
+    <string name="contactPickerActivityTitle" msgid="4301062192337417640">"Vælg en kontakt"</string>
     <string name="groupMemberPickerActivityTitle" msgid="1431750793695262522">"Vælg"</string>
-    <string name="header_entry_contact_list_adapter_header_title" msgid="2436981165830115659">"Opret ny kontaktperson"</string>
+    <string name="header_entry_contact_list_adapter_header_title" msgid="2436981165830115659">"Opret ny kontakt"</string>
     <string name="searchHint" msgid="8482945356247760701">"Søg i kontakter"</string>
     <string name="menu_addStar" msgid="2908478235715404876">"Føj til foretrukne"</string>
     <string name="menu_removeStar" msgid="5844227078364227030">"Fjern fra foretrukne"</string>
@@ -40,16 +40,16 @@
     <string name="menu_editGroup" msgid="6696843438454341063">"Fjern kontaktpersoner"</string>
     <string name="menu_renameGroup" msgid="7169512355179757182">"Omdøb etiket"</string>
     <string name="menu_deleteGroup" msgid="1126469629233412249">"Slet etiket"</string>
-    <string name="menu_addToGroup" msgid="3267409983764370041">"Tilføj kontaktperson"</string>
+    <string name="menu_addToGroup" msgid="3267409983764370041">"Tilføj kontakt"</string>
     <string name="menu_selectForGroup" msgid="3999234528229376098">"Vælg kontaktpersoner"</string>
     <string name="menu_addContactsToGroup" msgid="655196688840626483">"Tilføj kontaktpersoner"</string>
     <string name="menu_removeFromGroup" msgid="6720354305399961978">"Fjern fra etiket"</string>
     <string name="menu_new_group_action_bar" msgid="3127059073617415221">"Opret etiket"</string>
-    <string name="splitConfirmation" msgid="7342030840130187290">"Vil du ophæve sammenfletningen og opdele denne kontaktperson i flere kontaktpersoner?"</string>
+    <string name="splitConfirmation" msgid="7342030840130187290">"Vil du ophæve sammenfletningen og opdele denne kontakt i flere kontakter?"</string>
     <string name="splitConfirmation_positive_button" msgid="9129409098807939699">"Ophæv sammenfletningen"</string>
     <string name="splitConfirmationWithPendingChanges" msgid="7719062163511895696">"Vil du gemme de ændringer, du allerede har foretaget, og ophæve sammenfletningen og opdele kontaktpersonen i flere kontaktpersoner?"</string>
     <string name="splitConfirmationWithPendingChanges_positive_button" msgid="9073444264887244032">"Gem, og ophæv sammenfletningen"</string>
-    <string name="joinConfirmation" msgid="8262614843581924365">"Vil du gemme de ændringer, du allerede har foretaget, og sammenflette med den valgte kontaktperson?"</string>
+    <string name="joinConfirmation" msgid="8262614843581924365">"Vil du gemme de ændringer, du allerede har foretaget, og sammenflette med den valgte kontakt?"</string>
     <string name="joinConfirmation_positive_button" msgid="4573092849769149516">"Gem og flet"</string>
     <string name="contacts_linking_progress_bar" msgid="2846494347384549277">"Sammenfletter"</string>
     <string name="contacts_unlinking_progress_bar" msgid="5989310198163726929">"Ophæver sammenfletning"</string>
@@ -57,7 +57,7 @@
     <string name="menu_linkedContacts" msgid="400444389718855621">"Se sammenflettede kontaktpersoner"</string>
     <string name="menu_save" msgid="1727844363591825909">"Gem"</string>
     <string name="titleJoinContactDataWith" msgid="6825255752748313944">"Sammenflet kontaktpersoner"</string>
-    <string name="blurbJoinContactDataWith" msgid="5864256698061641841">"Vælg den kontaktperson, der skal knyttes til <xliff:g id="NAME">%s</xliff:g>:"</string>
+    <string name="blurbJoinContactDataWith" msgid="5864256698061641841">"Vælg den kontakt, der skal knyttes til <xliff:g id="NAME">%s</xliff:g>:"</string>
     <string name="separatorJoinAggregateSuggestions" msgid="2831414448851313345">"Foreslåede kontakter"</string>
     <string name="separatorJoinAggregateAll" msgid="7939932265026181043">"Alle kontakter"</string>
     <string name="contactsJoinedNamedMessage" msgid="1299418263439579455">"<xliff:g id="NAME">%s</xliff:g> er sammenflettet"</string>
@@ -70,33 +70,33 @@
       <item quantity="other">Kontaktpersonerne blev slettet</item>
     </plurals>
     <plurals name="contacts_count" formatted="false" msgid="8696793457340503668">
-      <item quantity="one"><xliff:g id="COUNT_1">%d</xliff:g> kontaktperson</item>
-      <item quantity="other"><xliff:g id="COUNT_1">%d</xliff:g> kontaktpersoner</item>
+      <item quantity="one"><xliff:g id="COUNT_1">%d</xliff:g> kontakt</item>
+      <item quantity="other"><xliff:g id="COUNT_1">%d</xliff:g> kontakter</item>
     </plurals>
     <plurals name="contacts_count_with_account" formatted="false" msgid="7402583111980220575">
-      <item quantity="one"><xliff:g id="COUNT_2">%1$d</xliff:g> kontaktperson · <xliff:g id="ACCOUNT_3">%2$s</xliff:g></item>
-      <item quantity="other"><xliff:g id="COUNT_2">%1$d</xliff:g> kontaktpersoner · <xliff:g id="ACCOUNT_3">%2$s</xliff:g></item>
+      <item quantity="one"><xliff:g id="COUNT_2">%1$d</xliff:g> kontakt · <xliff:g id="ACCOUNT_3">%2$s</xliff:g></item>
+      <item quantity="other"><xliff:g id="COUNT_2">%1$d</xliff:g> kontakter · <xliff:g id="ACCOUNT_3">%2$s</xliff:g></item>
     </plurals>
     <string name="title_from_google" msgid="4664084747121207202">"Fra Google"</string>
     <string name="title_from_other_accounts" msgid="8307885412426754288">"Fra <xliff:g id="ACCOUNT">%s</xliff:g>"</string>
     <string name="menu_set_ring_tone" msgid="8728345772068064946">"Angiv ringetone"</string>
     <string name="menu_redirect_calls_to_vm" msgid="83331924006712755">"Omdiriger til telefonsvarer"</string>
     <string name="menu_unredirect_calls_to_vm" msgid="8725542534701811737">"Træk tilbage fra telefonsvarer"</string>
-    <string name="readOnlyContactWarning" msgid="7514475664695089411">"Denne kontaktperson er skrivebeskyttet. Vedkommende kan ikke slettes, men kun skjules."</string>
-    <string name="readOnlyContactWarning_positive_button" msgid="8914797212167683673">"Skjul kontaktperson"</string>
-    <string name="readOnlyContactDeleteConfirmation" msgid="4717023505914239809">"Data om denne kontaktperson fra skrivebeskyttede konti bliver skjult, men ikke slettet."</string>
-    <string name="single_delete_confirmation" msgid="3106905986948679720">"Vil du slette denne kontaktperson?"</string>
+    <string name="readOnlyContactWarning" msgid="7514475664695089411">"Denne kontakt er skrivebeskyttet. Vedkommende kan ikke slettes, men kun skjules."</string>
+    <string name="readOnlyContactWarning_positive_button" msgid="8914797212167683673">"Skjul kontakt"</string>
+    <string name="readOnlyContactDeleteConfirmation" msgid="4717023505914239809">"Data om denne kontakt fra skrivebeskyttede konti bliver skjult, men ikke slettet."</string>
+    <string name="single_delete_confirmation" msgid="3106905986948679720">"Vil du slette denne kontakt?"</string>
     <string name="batch_delete_confirmation" msgid="3984346060324014108">"Vil du slette de valgte kontaktpersoner?"</string>
     <string name="batch_delete_read_only_contact_confirmation" msgid="8411117621035639964">"Kontaktpersoner fra dine skrivebeskyttede konti kan ikke slettes, men de kan skjules."</string>
     <string name="batch_delete_multiple_accounts_confirmation" msgid="8207205649127030030">"De kontaktpersoner, du vil slette, indeholder oplysninger fra flere konti. Oplysninger fra skrivebeskyttede konti skjules fremfor at blive slettet."</string>
-    <string name="multipleContactDeleteConfirmation" msgid="5235324124905653550">"Sletning af denne kontaktperson sletter oplysninger fra flere konti."</string>
-    <string name="deleteConfirmation" msgid="3512271779086656043">"Vil du slette denne kontaktperson?"</string>
+    <string name="multipleContactDeleteConfirmation" msgid="5235324124905653550">"Sletning af denne kontakt sletter oplysninger fra flere konti."</string>
+    <string name="deleteConfirmation" msgid="3512271779086656043">"Vil du slette denne kontakt?"</string>
     <string name="deleteConfirmation_positive_button" msgid="7857888845028586365">"Slet"</string>
     <string name="invalidContactMessage" msgid="8215051456181842274">"Kontaktpersonen findes ikke."</string>
     <string name="createContactShortcutSuccessful_NoName" msgid="8831303345367275472">"Kontaktpersonen blev føjet til startskærmen."</string>
     <string name="createContactShortcutSuccessful" msgid="953651153238790069">"<xliff:g id="NAME">%s</xliff:g> blev føjet til startskærmen."</string>
     <string name="photoPickerNotFoundText" msgid="2052542573853433204">"Der er ingen tilgængelige billeder på enheden."</string>
-    <string name="attach_photo_dialog_title" msgid="5599827035558557169">"Billede af kontaktperson"</string>
+    <string name="attach_photo_dialog_title" msgid="5599827035558557169">"Billede af kontakt"</string>
     <string name="customLabelPickerTitle" msgid="1081475101983255212">"Navn på tilpasset etiket"</string>
     <string name="removePhoto" msgid="4898105274130284565">"Fjern billede"</string>
     <string name="noContacts" msgid="2228592924476426108">"Din liste over kontaktpersoner er tom"</string>
@@ -147,7 +147,7 @@
     <string name="label_name_dialog_hint" msgid="1276051790427638142">"Etiketnavn"</string>
     <string name="audio_chat" msgid="2535716629358298691">"Talechat"</string>
     <string name="video_chat" msgid="1872255818640336072">"Videochat"</string>
-    <string name="account_type_format" msgid="718948015590343010">"<xliff:g id="SOURCE">%1$s</xliff:g> kontaktperson"</string>
+    <string name="account_type_format" msgid="718948015590343010">"<xliff:g id="SOURCE">%1$s</xliff:g> kontakt"</string>
     <string name="google_account_type_format" msgid="5283997303922067997">"<xliff:g id="SOURCE">%1$s</xliff:g>-konto"</string>
     <string name="take_photo" msgid="7496128293167402354">"Tag billede"</string>
     <string name="take_new_photo" msgid="7341354729436576304">"Tag nyt billede"</string>
@@ -159,7 +159,7 @@
     <string name="cancel_button_content_description" msgid="1288652456274531846">"Annuller"</string>
     <string name="back_arrow_content_description" msgid="4355362760545735065">"Tilbage"</string>
     <string name="action_menu_back_from_edit_select" msgid="6435476408621731420">"luk"</string>
-    <string name="aggregation_suggestion_edit_dialog_message" msgid="6549585283910518095">"Vil du redigere den valgte kontaktperson? Dine indtastninger kopieres."</string>
+    <string name="aggregation_suggestion_edit_dialog_message" msgid="6549585283910518095">"Vil du redigere den valgte kontakt? Dine indtastninger kopieres."</string>
     <string name="contact_directory_description" msgid="683398073603909119">"Indeks <xliff:g id="TYPE">%1$s</xliff:g>"</string>
     <string name="activity_title_settings" msgid="5464130076132770781">"Indstillinger"</string>
     <string name="menu_settings" msgid="377929915873428211">"Indstillinger"</string>
@@ -189,17 +189,17 @@
     <string name="contact_editor_prompt_zero_accounts" msgid="6648376557574360096">"Brug et øjeblik på at tilføje en konto, som sikkerhedskopierer dine kontaktpersoner til Google."</string>
     <string name="contact_editor_prompt_one_account" msgid="3087691056345099310">"Nye kontakter gemmes på <xliff:g id="ACCOUNT_NAME">%1$s</xliff:g>."</string>
     <string name="contact_editor_prompt_multiple_accounts" msgid="8565761674283473549">"Vælg en standardkonto til nye kontaktpersoner:"</string>
-    <string name="contact_editor_title_new_contact" msgid="3042788718983070111">"Opret ny kontaktperson"</string>
-    <string name="contact_editor_title_existing_contact" msgid="4898475703683187798">"Rediger kontaktperson"</string>
+    <string name="contact_editor_title_new_contact" msgid="3042788718983070111">"Opret ny kontakt"</string>
+    <string name="contact_editor_title_existing_contact" msgid="4898475703683187798">"Rediger kontakt"</string>
     <string name="contact_editor_title_read_only_contact" msgid="2443496574528842237">"Kun visning"</string>
-    <string name="contact_editor_pick_raw_contact_to_edit_dialog_title" msgid="4186699385645902152">"Vælg en kontaktperson at redigere"</string>
+    <string name="contact_editor_pick_raw_contact_to_edit_dialog_title" msgid="4186699385645902152">"Vælg en kontakt at redigere"</string>
     <string name="contact_editor_pick_linked_contact_dialog_title" msgid="4618465940423857694">"Sammenflettede kontaktpersoner"</string>
     <string name="contact_editor_add_linked_contact" msgid="426142748048816297">"Tilføj"</string>
     <string name="contact_editor_unlink_contacts" msgid="4525214336177236653">"Ophæv sammenfletning"</string>
     <string name="add_account" msgid="8201790677994503186">"Tilføj konto"</string>
     <string name="add_new_account" msgid="5748627740680940264">"Tilføj ny konto"</string>
     <string name="menu_export_database" msgid="2659719297530170820">"Eksportér databasefiler"</string>
-    <string name="action_menu_add_new_contact_button" msgid="5506832825256203208">"Opret ny kontaktperson"</string>
+    <string name="action_menu_add_new_contact_button" msgid="5506832825256203208">"Opret ny kontakt"</string>
     <string name="expanding_entry_card_view_see_more" msgid="3779194067124758079">"Se mere"</string>
     <string name="expanding_entry_card_view_see_less" msgid="5344160551629714168">"Se mindre"</string>
     <string name="recent_card_title" msgid="8982782042698001695">"Seneste"</string>
@@ -232,15 +232,15 @@
     <string name="editor_more_fields" msgid="3630987771304393421">"Flere felter"</string>
     <string name="editor_change_photo_content_description" msgid="820879708069864340">"Skift kontaktpersonens billede"</string>
     <string name="editor_add_photo_content_description" msgid="4194479313465418120">"Tilføj et billede af kontaktpersonen"</string>
-    <string name="editor_contact_photo_content_description" msgid="8571975622926162369">"Billede af kontaktperson"</string>
+    <string name="editor_contact_photo_content_description" msgid="8571975622926162369">"Billede af kontakt"</string>
     <string name="editor_failed_to_load" msgid="1623639078580475818">"Redigeringsværktøjet kunne ikke åbnes."</string>
     <string name="editor_account_selector_title" msgid="4426337993307015833">"Gemmes på"</string>
     <string name="editor_account_selector_read_only_title" msgid="4137969650184457126">"Kontaktoplysningerne fra <xliff:g id="ACCOUNT">%s</xliff:g> kan ikke redigeres"</string>
     <string name="quickcontact_suggestion_link_button" msgid="3244619714781727946">"SAMMENFLET KONTAKTPERSONER"</string>
     <string name="quickcontact_suggestion_cancel_button" msgid="8236954313106630862">"ANNULLER"</string>
-    <string name="suggestion_card_this_contact_title" msgid="3039457405374454914">"Denne kontaktperson"</string>
+    <string name="suggestion_card_this_contact_title" msgid="3039457405374454914">"Denne kontakt"</string>
     <string name="suggestion_card_duplicates_title" msgid="9107788743178980902">"Mulige dubletter"</string>
-    <string name="suggestion_card_help_message" msgid="4474061044387181093">"Disse kontaktpersoner kan være den samme person. Du kan sammenflette dem til én enkelt kontaktperson."</string>
+    <string name="suggestion_card_help_message" msgid="4474061044387181093">"Disse kontakter kan være den samme person. Du kan sammenflette dem til én enkelt kontakt."</string>
     <string name="editor_delete_view_description" msgid="8583095381562991959">"Slet <xliff:g id="DATA_TYPE">%1$s </xliff:g><xliff:g id="DATA_KIND">%2$s</xliff:g>"</string>
     <string name="editor_delete_view_description_short" msgid="7335518371270844912">"Slet <xliff:g id="DATA_KIND">%s</xliff:g>"</string>
     <string name="menu_assistant" msgid="5760693735722052454">"Forslag"</string>
@@ -264,7 +264,7 @@
     <string name="call_fax_work" msgid="7467763592359059243">"Ring til arbejdsfax"</string>
     <string name="call_fax_home" msgid="8342175628887571876">"Ring til hjemmefax"</string>
     <string name="call_pager" msgid="9003902812293983281">"Ring til personsøger"</string>
-    <string name="call_other" msgid="8563753966926932052">"Opkald"</string>
+    <string name="call_other" msgid="8563753966926932052">"Ring op"</string>
     <string name="call_callback" msgid="1910165691349426858">"Ring til tilbagekald"</string>
     <string name="call_car" msgid="3280537320306436445">"Ring til bil"</string>
     <string name="call_company_main" msgid="6105120947138711257">"Ring til arbejde (hovednummer)"</string>
@@ -383,7 +383,7 @@
     <string name="list_filter_all_accounts" msgid="8908683398914322369">"Alle kontaktpersoner"</string>
     <string name="list_filter_all_starred" msgid="5031734941601931356">"Stjernemarkerede"</string>
     <string name="list_filter_customize" msgid="4789963356004169321">"Tilpas"</string>
-    <string name="list_filter_single" msgid="5871400283515893087">"Kontaktperson"</string>
+    <string name="list_filter_single" msgid="5871400283515893087">"Kontakt"</string>
     <string name="display_ungrouped" msgid="6885954210243119591">"Alle andre kontaktpersoner"</string>
     <string name="display_all_contacts" msgid="2031647544742889505">"Alle kontaktpersoner"</string>
     <string name="menu_sync_remove" msgid="3266725887008450161">"Fjern synkroniseringsgruppe"</string>
@@ -396,12 +396,12 @@
     <string name="import_from_sim" msgid="6912071635295799131">"SIM-kort"</string>
     <string name="import_from_sim_summary_fmt" msgid="5169032449686348118">"SIM-kort <xliff:g id="SIM_NAME">%1$s</xliff:g>"</string>
     <plurals name="import_from_sim_secondary_contact_count_fmt" formatted="false" msgid="5821095773211149295">
-      <item quantity="one">%1$d kontaktperson</item>
-      <item quantity="other">%1$d kontaktpersoner</item>
+      <item quantity="one">%1$d kontakt</item>
+      <item quantity="other">%1$d kontakter</item>
     </plurals>
     <plurals name="import_from_sim_secondary_template" formatted="false" msgid="2484832025802907060">
-      <item quantity="one"><xliff:g id="COUNT_2">^1</xliff:g> kontaktperson • <xliff:g id="PHONE_NUMBER_3">^2</xliff:g></item>
-      <item quantity="other"><xliff:g id="COUNT_2">^1</xliff:g> kontaktpersoner • <xliff:g id="PHONE_NUMBER_3">^2</xliff:g></item>
+      <item quantity="one"><xliff:g id="COUNT_2">^1</xliff:g> kontakt • <xliff:g id="PHONE_NUMBER_3">^2</xliff:g></item>
+      <item quantity="other"><xliff:g id="COUNT_2">^1</xliff:g> kontakter • <xliff:g id="PHONE_NUMBER_3">^2</xliff:g></item>
     </plurals>
     <string name="import_from_vcf_file" msgid="8662528435646418203">".vcf-fil"</string>
     <string name="nothing_to_import_message" msgid="1651921906873335656">"Der er intet at importere"</string>
@@ -465,7 +465,7 @@
     <string name="share_contacts_failure" msgid="1216431977330560559">"Kontaktpersonerne kunne ikke deles."</string>
     <string name="dialog_export" msgid="1628001095187741417">"Eksportér kontaktpersoner"</string>
     <string name="dialog_import" msgid="2423592905927819672">"Importér kontaktpersoner fra"</string>
-    <string name="share_error" msgid="948429331673358107">"Denne kontaktperson kan ikke deles."</string>
+    <string name="share_error" msgid="948429331673358107">"Denne kontakt kan ikke deles."</string>
     <string name="no_contact_to_share" msgid="1276397530378323033">"Der er ingen kontaktpersoner, der kan deles."</string>
     <string name="menu_contacts_filter" msgid="2165153460860262501">"Viste kontaktpersoner"</string>
     <string name="activity_title_contacts_filter" msgid="8275542497615516969">"Viste kontaktpersoner"</string>
@@ -509,8 +509,8 @@
     <string name="sim_import_empty_message" msgid="7743815244380189651">"Der er ingen kontaktpersoner på dit SIM-kort"</string>
     <string name="sim_import_contact_exists_toast" msgid="1503743663717316732">"Kontaktpersonen findes allerede på din liste over kontakter"</string>
     <plurals name="sim_import_success_toast_fmt" formatted="false" msgid="8572156521110906443">
-      <item quantity="one"><xliff:g id="COUNT">%d</xliff:g> kontaktperson på SIM-kortet blev importeret</item>
-      <item quantity="other"><xliff:g id="COUNT">%d</xliff:g> kontaktpersoner på SIM-kortet blev importeret</item>
+      <item quantity="one"><xliff:g id="COUNT">%d</xliff:g> kontakt på SIM-kortet blev importeret</item>
+      <item quantity="other"><xliff:g id="COUNT">%d</xliff:g> kontakter på SIM-kortet blev importeret</item>
     </plurals>
     <string name="sim_import_failed_toast" msgid="5559267299793622705">"Kontaktpersonerne på SIM-kortet kan ikke importeres"</string>
     <string name="sim_import_title" msgid="2511154832820812822">"Importér fra SIM-kort"</string>
diff --git a/res/values-iw/strings.xml b/res/values-iw/strings.xml
index a6159ef71..774c8adad 100644
--- a/res/values-iw/strings.xml
+++ b/res/values-iw/strings.xml
@@ -177,7 +177,7 @@
     <string name="non_phone_add_to_contacts" msgid="6590985286250471169">"הוסף לאנשי הקשר"</string>
     <string name="non_phone_close" msgid="7608506439725515667">"סגור"</string>
     <string name="date_year_toggle" msgid="7122002148518724139">"ציין שנה"</string>
-    <string name="contacts_unavailable_add_account" msgid="4347232421410561500">"הוסף חשבון"</string>
+    <string name="contacts_unavailable_add_account" msgid="4347232421410561500">"הוספת חשבון"</string>
     <string name="contacts_unavailable_import_contacts" msgid="3182801738595937144">"ייבא"</string>
     <string name="create_group_item_label" msgid="3263064599743742865">"צור חדשה…"</string>
     <string name="delete_group_dialog_message" msgid="335713829185261371">"למחוק את התווית \"<xliff:g id="GROUP_LABEL">%1$s</xliff:g>\"? (אנשי הקשר המסומנים בתווית לא יימחקו)."</string>
@@ -204,7 +204,7 @@
     <string name="contact_editor_pick_linked_contact_dialog_title" msgid="4618465940423857694">"אנשי קשר מקושרים"</string>
     <string name="contact_editor_add_linked_contact" msgid="426142748048816297">"הוסף"</string>
     <string name="contact_editor_unlink_contacts" msgid="4525214336177236653">"בטל את הקישור"</string>
-    <string name="add_account" msgid="8201790677994503186">"הוסף חשבון"</string>
+    <string name="add_account" msgid="8201790677994503186">"הוספת חשבון"</string>
     <string name="add_new_account" msgid="5748627740680940264">"הוסף חשבון חדש"</string>
     <string name="menu_export_database" msgid="2659719297530170820">"ייצוא קובצי מסד נתונים"</string>
     <string name="action_menu_add_new_contact_button" msgid="5506832825256203208">"איש קשר חדש"</string>
diff --git a/res/values-ne/strings.xml b/res/values-ne/strings.xml
index fca914758..4f102dc4c 100644
--- a/res/values-ne/strings.xml
+++ b/res/values-ne/strings.xml
@@ -460,7 +460,7 @@
     <string name="settings_my_info_title" msgid="1534272456405343119">"मेरा बारेमा जानकारी"</string>
     <string name="set_up_profile" msgid="7370213843590143771">"आफ्नो प्रोफाइल सेटअप गर्नुहोस्"</string>
     <string name="setting_about" msgid="7014388749752042863">"सम्पर्कहरूका बारेमा"</string>
-    <string name="share_favorite_contacts" msgid="4280926751003081042">"मनपर्ने सम्पर्कहरू आदान प्रदान गर्नुहोस्"</string>
+    <string name="share_favorite_contacts" msgid="4280926751003081042">"मन पर्ने सम्पर्कहरू आदान प्रदान गर्नुहोस्"</string>
     <string name="share_contacts" msgid="8109287987498711664">"सबै सम्पर्क आदान प्रदान गर्ने"</string>
     <string name="share_contacts_failure" msgid="1216431977330560559">"सम्पर्कहरू आदान प्रदान गर्न सकिएन।"</string>
     <string name="dialog_export" msgid="1628001095187741417">"सम्पर्क ठेगानाहरू निर्यात गर्नुहोस्"</string>
@@ -472,7 +472,7 @@
     <string name="custom_list_filter" msgid="9048525797111024204">"दृश्यलाई आफू अनुकूल पार्नुहोस्"</string>
     <string name="menu_custom_filter_save" msgid="2679793632208086460">"सुरक्षित गर्ने"</string>
     <string name="hint_findContacts" msgid="7128627979899070325">"सम्पर्क ठेगानाहरू खोज्नुहोस्"</string>
-    <string name="contactsFavoritesLabel" msgid="8417039765586853670">"मनपर्ने"</string>
+    <string name="contactsFavoritesLabel" msgid="8417039765586853670">"मन पर्ने"</string>
     <string name="menu_import" msgid="6107961135813836467">"आयात गर्नुहोस्"</string>
     <string name="menu_export" msgid="2658783911863503902">"निर्यात गर्नुहोस्"</string>
     <string name="menu_blocked_numbers" msgid="5272951629083025995">"रोक लगाइएका नम्बरहरू"</string>
diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 4ec351b95..cdf0ff01e 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -175,8 +175,8 @@
     <string name="delete_group_dialog_message" msgid="335713829185261371">"Ungependa kufuta lebo iitwayo \"<xliff:g id="GROUP_LABEL">%1$s</xliff:g>\"? (Anwani zenyewe hazitafutwa.)"</string>
     <string name="toast_join_with_empty_contact" msgid="1215465657839085613">"Andika jina ya anwani kabla ya kuiunganisha na nyingine."</string>
     <string name="copy_text" msgid="3257145021583508761">"Nakili kwenye ubao wa kunakili"</string>
-    <string name="set_default" msgid="4417505153468300351">"Weka chaguo msingi"</string>
-    <string name="clear_default" msgid="7193185801596678067">"Ondoa chaguo msingi"</string>
+    <string name="set_default" msgid="4417505153468300351">"Weka chaguomsingi"</string>
+    <string name="clear_default" msgid="7193185801596678067">"Ondoa chaguomsingi"</string>
     <string name="toast_text_copied" msgid="5143776250008541719">"Maandishi yamenakiliwa"</string>
     <string name="cancel_confirmation_dialog_message" msgid="9008214737653278989">"Una uhakika hutaki kuhifadhi mabadiliko uliyofanya?"</string>
     <string name="cancel_confirmation_dialog_cancel_editing_button" msgid="3057023972074640671">"Tupa"</string>
@@ -456,7 +456,7 @@
     <string name="display_options_view_given_name_first" msgid="3616004640258761473">"Anza kwa jina la kwanza"</string>
     <string name="display_options_view_family_name_first" msgid="956445100777296467">"Anza kwa jina la mwisho"</string>
     <string name="settings_accounts" msgid="350219740670774576">"Akaunti"</string>
-    <string name="default_editor_account" msgid="699591683362420991">"Akaunti chaguo msingi ya kuhifadhi anwani mpya"</string>
+    <string name="default_editor_account" msgid="699591683362420991">"Akaunti chaguomsingi ya kuhifadhi anwani mpya"</string>
     <string name="settings_my_info_title" msgid="1534272456405343119">"Maelezo yangu"</string>
     <string name="set_up_profile" msgid="7370213843590143771">"Weka wasifu wako"</string>
     <string name="setting_about" msgid="7014388749752042863">"Kuhusu Anwani"</string>
-- 
2.17.1

